python_bindings/tests/engine_test.py

```diff
@@ -85,3 +91,102 @@ def test_game_engine_send_batch_returns_failures_when_communicator_closed(
         expected = requests[idx].GameID
         actual = resp.GameID()
         assert actual == expected
+
+
+def test_game_engine_send_get_remaining_tiles_batch_returns_remaining_tiles(
+    tmp_path: Path
+) -> None:
+    t1 = tiletemplates.MonasteryWithSingleRoad()
+    t2 = tiletemplates.RoadsTurn()
+    tiles = [t1, t2, t1]
+    tile_set = TileSet(
+        StartingTile=tiletemplates.SingleCityEdgeStraightRoads(),
+        Tiles=Slice_tiles_Tile(tiles),
+    )
+
+    engine = StartGameEngine(1, str(tmp_path))
+    game = engine.GenerateGame(tile_set)
```

Comment:
Is there a way to clone a game state? Mostly to allow for simulating deeper branches of a specific game.

Reply:
Nope. I can write a new request for cloning a game with the given ID that would return the new ID, should be simple to do.

Reply:
Yeah, that would be nice since currently game needs to be replayed for every expansion of the game tree.